tv_apps/smart-system/js/system_window.js

```diff
@@ -0,0 +1,49 @@
+/* global AudioChannelController */
+/* global BaseModule */
+'use strict';
+
+(function() {
+  /**
+   * SystemWindow is a placeholder for the system app's mozbrowser iframe
+   * but with no actual UI.
+   * We will use mozSystemWindowChromeEvent/mozContentEvent to communicate
+   * with shell.js to use the actual mozbrowser iframe of the system app.
+   */
+  var SystemWindow = function() {};
```

Comment:
Yes, I think we could do that, although we don't have case that playing sounds from System app. We could verify this part when we have a case.